extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php

```diff
@@ -115,30 +116,35 @@ public function getMenuNodes() {
 			$hubsNodes = [];
 		}
 
-		// Link to WAM - Top Communities
 		$exploreDropdownLinks[] = [
-			'text' => $WAMLinkLabel->plain(),
-			'textEscaped' => $WAMLinkLabel->escaped(),
-			'href' => $this->getWAMLinkForLang( $wgLang->getCode() ),
-			'trackingLabel' => 'top-communities',
+			'text' => $exploreWikisLabel->plain(),
+			'textEscaped' => $exploreWikisLabel->escaped(),
+			'href' => wfMessage('global-navigation-explore-wikis-link')->plain(),
+			'trackingLabel' => 'explore-wikis',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "trackingLabel" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)